_usage/locales.md

```diff
@@ -285,7 +292,7 @@ first release.
 When a single-byte character (half-width) and the same character
 represented as a double-byte character (full-width) are treated
 differently, the collation is called *width-sensitive (WS)*. WS
-collations with the same name as SQL Server won\'t be provided by
+collations with the same name as SQL Server won't be provided by
```

Comment:
maybe we want to refer to the 'current release' instead of the 'first release'

Reply:
I agree with Jim - we can remove the limitation when a version changes the behavior...

Reply:
Done in 2ed7c3f504fbf9e26aa9866b490ab841662f0bdd. I did that change in three places.